pkgs/development/libraries/gobject-introspection/wrapper.nix

```diff
@@ -28,5 +28,20 @@ gobject-introspection-unwrapped.overrideAttrs (previousAttrs: {
       chmod +x "$dev/bin/g-ir-compiler"
       chmod +x "$dev/bin/g-ir-scanner"
     )
+  ''
+  # when cross-compiling and using the wrapper then when a package looks up the g_ir_X
+  # variable with pkg-config they'll get the host version which can't be run
+  # switch the variables to use g_ir_X from path instead of an absolute path
```

Comment:
The comment is outdated.

Reply:
join the cross-compilation room on matrix https://matrix.to/#/#cross-compiling:nixos.org

Reply:
should be up to date now